Dockerfile

```diff
@@ -7,19 +7,20 @@ MAINTAINER wurstmeister
 
 ENV KAFKA_VERSION=$kafka_version \
     SCALA_VERSION=$scala_version \
-    KAFKA_HOME=/opt/kafka \
-    PATH=${PATH}:${KAFKA_HOME}/bin
+    KAFKA_HOME=/opt/kafka 
+
+ENV PATH=${PATH}:${KAFKA_HOME}/bin
```

Comment:
With the recent test updates, you should be able to add a test for this now. i.e. add a `test/xxx-test-path.kafka.sh` script that does something like:

```
echo "Checking PATH '$PATH'"
[[ "$PATH" ~= "/opt/kafka/bin" ]]
```

This will help make sure it doesn't get removed in future. Thanks

Reply:
done

Reply:
Sorry, looks like a typo in my suggested ad hoc test code. Should be '=~' test (not the invalid  '~=')

test/Readme.md should give instructions on how to execute tests locally